solr/core/src/java/org/apache/solr/pkg/PackageLoader.java

```diff
@@ -301,6 +295,32 @@ public String toString() {
       }
     }
   }
+  static class PackageResourceLoader extends SolrResourceLoader {
+
+    PackageResourceLoader(String name, List<Path> classpath, Path instanceDir, ClassLoader parent) {
+      super(name, classpath, instanceDir, parent);
+    }
+
+    @Override
+    public <T> boolean addToCoreAware(T obj) {
+      //do not do anything
+      //this class is
```

Comment:
incomplete comment?  Could use an explanation as to why. SolrCoreAware isn't handled here.